app/scripts/components/common/shadow-scrollbar-improved.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import { ShadowScrollbar } from '@devseed-ui/shadow-scrollbar';
 
+// Moving away from this component because of the issue
+// https://github.com/NASA-IMPACT/veda-ui/issues/682#issuecomment-1747334303
```

Comment:
I think you can go ahead and remove the file. Juste use this comment as the commit message.